app/src/main/java/org/oppia/app/customview/interaction/EditTextBindingAdapters.java

```diff
@@ -0,0 +1,13 @@
+package org.oppia.app.customview.interaction;
```

Comment:
Nit: please make this a Kotlin file instead.

Reply:
Done